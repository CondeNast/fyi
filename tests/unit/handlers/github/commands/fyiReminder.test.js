const fyiReminderEvent = require('../../../fixtures/fyi-reminder')
const models = require('../../../../../src/models')

let app
let github

describe('Arch Bot', () => {
  beforeEach(() => {
    ({app, github} = require('../../../setup.js'))
  })

  describe('FYI Reminder', () => {
    it('Posts Comment for Issue in Repo Repository', async () => {
      await app.receive(fyiReminderEvent)
      expect(github.issues.createComment).toMatchSnapshot()
    })
  })

  afterAll(async () => {
    await models.sequelize.close()
  })
})
