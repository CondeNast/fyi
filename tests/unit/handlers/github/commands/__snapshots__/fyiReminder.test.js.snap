// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arch Bot FYI Reminder Posts Comment for Issue in Repo Repository 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      Object {
        "body": "Reminder: Complete the requested FYI",
        "number": 1,
        "owner": "choosenearme",
        "repo": "super-guacamole",
      },
    ],
  ],
}
`;
